src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,97 @@
+package main.java.Duke;
+
+import main.java.Duke.Task.Deadline;
+import main.java.Duke.Task.Event;
+import main.java.Duke.Task.Task;
+import main.java.Duke.Task.toDo;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class Storage {
+    public String filePath;
+    public ArrayList<Task> arr;
```

Comment:
It is important to ensure that your class attributes are labelled as private. I also noticed some other attributes in other classes without access modifiers or having the wrong ones attached to them. Do take note of this!